### DISCUSSION TITLE
[Feature Request] add a trigger action for denied PR

### DISCUSSION THREAD

**Discussion by HeidiRechek** [jenkinsci/bitbucket-push-and-pull-request-plugin#213] [Ideas]
There is actions that should happen if a PR is declined.  They webhook has the ability to push the Declined event, but the jenkins plugin doesn't have a action for that to trigger on. 

Our use case is that on PR Create we allocate an on demand sandbox, on Merge it is cleaned up, on decline, it is currently orphaned. We want to automate cleanup on "declined:

[a comment by an automated account is not shown]

**Comment by y0zg** [comment]
@HeidiRechek just wondering whether you managed to solve the issue
In my case I don't need PRs to be declined whenever jenkins job is failed. Any ideas how to do this?

**Comment by HeidiRechek** [comment]
no solve inplace was waiting on this update

**Comment by cdelmonte-zg** [comment]
Hallo @HeidiRechek This feature will added in the next release next week, I must solve some merge conflicts and check once again if all is still working as aspected.
